bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -28,9 +28,6 @@
 {% block string_data %}
 data-devtools-title="{{ _('Developer Tools') }}"
 data-devtools-text="{{ _('The most complete browser made just for developers, Firefox Developer Edition has every dev tool you’ll need built right in.') }}"
-data-next-webide="{{ _('Next: WebIDE') }}"
```

Comment:
Removing this will likely break some JS in the page. Best check that doesn't need updating also.